all-clouds/public-images/index.rst

```diff
@@ -0,0 +1,54 @@
+Ubuntu public images
+====================
+
+**Canonical produce generic (generic kernel) cloud images, LXD images (rootfs tarballs) and KVM optimised cloud images (KVM kernel)**. These images are public (unlike other cloud-specific images) and are available on `cloud-images.ubuntu.com <http://cloud-images.ubuntu.com>`_.
```

Comment:
leave for now, but the KVM kernel is deprecated. I've got docs in the works (that i'm staring at now)

Reply:
Noted.